src/GraknClient.jl

```diff
@@ -76,29 +83,59 @@ include("core/CoreClient.jl")
 include("core/CoreSession.jl")
 include("core/CoreTransaction.jl")
 
+# part of common section -- place because of general grpc resulthandling
+include("common/exception/gRPC_Result_Handling.jl")
+
 
 export GraknCoreBlockingClient, GraknClientException
```

Comment:
Have these not been renamed yet? GraknCore -> TypeDBCore, GraknClient -> TypeDBClient?